src/Client/ClientBase.cpp

```diff
@@ -1366,12 +1366,19 @@ void ClientBase::runInteractive()
 
         for (const auto& [alias, command] : backslash_aliases)
         {
-            if (input.starts_with(alias))
+            auto it = std::search(input.begin(), input.end(), alias.begin(), alias.end());
+            if (it != input.end() && std::all_of(input.begin(), it, isWhitespaceASCII))
```

Comment:
In `ClientBase::processQueryText()` we have already trimmed text, so it may have sense to  process that trimmed string with.